pkg/deploy/join.go

```diff
@@ -17,16 +17,16 @@ package deploy
 type WithJoinOptions struct {
 	PreferredServer               bool
 	ForceUDPEncaps                bool
-	NatTraversal                  bool
+	NATTraversal                  bool
```

Comment:
Again, not sure if renaming flags is ideal to do without any warning.

Reply:
This isn’t renaming a flag, is it? The flag names are determined by the flag setup code (`PersistentFlags().StringVar()` etc.).

Reply:
Same

Reply:
yeah this is just the struct field which is new.